### DISCUSSION TITLE
Error decrypting message: Session key decryption failed

### DISCUSSION THREAD

**Discussion by abuhmeedan** [openpgpjs/openpgpjs#1429] [Q&A]
Facing the below issue while decrypting message (V: 5.0.0)

 Error: Error decrypting message: Session key decryption failed.
 
 I'm using the sample code from the documentation

**Comment by larabr** [comment]
Hi @abuhmeedan ,
there can be a number of explanations for the problem.
Have you checked the existing discussions/issues on the matter? (https://github.com/openpgpjs/openpgpjs/issues/1126, https://github.com/openpgpjs/openpgpjs/discussions/1306, etc.)
If none of those apply to your case, please post more information about e.g. whether decryption works in other libraries. Or share the encrypted key and message.
